test/controllers/bigbluebutton_api_controller_test.rb

```diff
@@ -653,6 +653,19 @@ class BigBlueButtonApiControllerTest < ActionDispatch::IntegrationTest
     assert_redirected_to encode_bbb_uri('join', server1.url, server1.secret, params).to_s
   end
 
+  test 'redirects user to the corrent join url with only permitted params for join' do
```

Comment:
For testing the controller methods, maybe you can stub the `Rails.configuration.x.create_exclude_params` method? That would solve the issue of being able to test multiple different settings at runtime.

You still also need a separate test to make sure that the parsing in `config/application/rb` produced the expected values.